### PR TITLE
Scroll long messages, fix for !hotseat missing @ bug

### PR DESCRIPTION
**Pull request by insin**
Wee demo (jerky as the GIF is only 30fps):

rpDuDwUTek

### CODE REVIEW

File: twitchpopups.htm [hunk 1]
```diff
@@ -21,32 +22,37 @@
     </script>
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="jquery-3.4.1.min.js"></script>
```
Comment (Rcjuk) [inline twitchpopups.htm:24]: I think this is subjective. Nothing wrong with jQuery.

File: twitchpopups.htm [hunk 2]
```diff
@@ -97,25 +105,23 @@ <h1 id="popuptext"></h1>
...
                     doAnimation();
                 } else if (commandName.startsWith("!delete")) {
                   deleteAnimation();
                 } else if (commandName.startsWith("!hotseat ")) {
                   hotseatison = true;
-                  $("#popupbox").show();
-                  hotseatuser = commandName.substr(10).toLowerCase();
-                  $("#popupbox").css({"background-color":hotseatBg}); 
-                  $("#popuptext").text(hotseatEmoji + " " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT " + hotseatEmoji);
+                  hotseatuser = commandName.replace(/!hotseat @?/, '').toLowerCase();
```
Comment (Rcjuk) [inline twitchpopups.htm:115]: Nice 👍

File: twitchpopups.htm (hunk 1 above)
Comment (insin) [reply twitchpopups.htm:24]: firefox_sDiYSljycS

I meant nothing by it, just removing unused code 👀

**Comment by insin** [comment]
Moving this to a different PR, as it's completely different from the original thing I submitted

**Pull request closed without merging**